configserver/src/main/java/com/yahoo/vespa/config/server/filedistribution/CombinedLegacyDistribution.java

```diff
@@ -65,7 +65,7 @@ private void startDownloadingFileReferences(String hostName, int port, Set<FileR
         request.parameters().add(new StringArray(fileReferences.stream().map(FileReference::value).toArray(String[]::new)));
         log.log(LogLevel.DEBUG, "Executing " + request.methodName() + " against " + target.toString());
         target.invokeSync(request, timeout);
-        if (request.isError()) {
+        if (request.isError() && request.errorCode() != 104) {
```

Comment:
No constant hinting to what the value 104 indicates ?

Reply:
Fixed